locales/en-US/messages.properties

```diff
@@ -112,7 +112,7 @@ snippetHTMLScript=Include a JavaScript file
 snippetHTMLScriptTitle=Add a Script
 snippetHTMLExternalStylesheet=Include a CSS file
 snippetHTMLExternalStylesheetTitle=Add an External Stylesheet
-snippetHTMLExternalStylesheetComment=Add the <link> tag to the <head> tag of your HTML file
+snippetHTMLExternalStyleSheetComment=Add the <link> tag to your HTML file, preferably to the <head> tag
```

Comment:
I'd recommnd `Add the <link> tag to your HTML file.  These usually go in the <head> tag`